examples/text_to_speech_synthesizer/README.md

```diff
@@ -0,0 +1,37 @@
+# Text to speech synthesis using WaveGlow & Tacotron2 model.
+
+**This example works only on NVIDIA CUDA device and not on CPU**
+
+We have used the following Waveglow/Tacotron2 model for this example: 
+
+https://pytorch.org/hub/nvidia_deeplearningexamples_waveglow/
+
+We have copied WaveGlow's model file from following github repo:
+https://github.com/NVIDIA/DeepLearningExamples/blob/master/PyTorch/SpeechSynthesis/Tacotron2/waveglow/model.py
+
+
+# Install pip dependencies using following commands
+
+```bash
+pip install numpy scipy unidecode inflect
+pip install librosa --user
+```
+
+# Serve the WaveGlow speech synthesis model on TorchServe
+
+ * Generate the model archive for waveglow speech synthesis model using following command
+ 
+    ```bash
+    ./create_mar.sh
+    ```
+   
+ * Register the model on TorchServe using the above model archive file and run digit recognition inference
+   
+    ```bash
+    mkdir model_store
+    mv waveglow_synthesizer.mar model_store/
+    torchserve --start --model-store model_store --models waveglow_synthesizer.mar
+    curl -X POST http://127.0.0.1:8080/predictions/waveglow_synthesizer -T sample_text.txt -o audio.wav
```

Comment:
@chauhang : We have created this example using the checkpoint files, for tacotron and waveglow model, and the inference code used by following example provided in following `PyTorch` blog :
https://pytorch.org/hub/nvidia_deeplearningexamples_waveglow/ 

However, this model doesn't work properly for bigger text size as input. I have created following GH issue on NVIDIA/DeeplearningExamples repo : https://github.com/NVIDIA/DeepLearningExamples/issues/497

Reply:
@harshbafna It will be good to add reference to [this](https://github.com/NVIDIA/DeepLearningExamples/issues/497) issue in the example readme

Reply:
What about the steps for downloading the audio file from python client using the TorchServe REST API? Is there any special handling needed?  It will be good to include a small python client code example

Reply:
Just added a sample python snippet in the ReadMe.

Reply:
Regarding the limits on the size of the text that can be converted to audio :

Apart from the model limitations, the data size in request and response is governed by max_request_size and max_response_size config parameters. 

These parameters are not documented in configuration.md and will be taken up as part of fix for #204 ,where we will be updating all the config params and their respective environment variable names.

Reply:
@chauhang : Please let us know, if we should merge this PR with documentation changes for https://github.com/NVIDIA/DeepLearningExamples/issues/497 and create another ticket for tracking the enhancement of waveglow_handler based on the solution from https://github.com/NVIDIA/DeepLearningExamples/issues/497

Reply:
@chauhang : Based on the comments on https://github.com/NVIDIA/DeepLearningExamples/issues/497 we will need do additional pre-processing and post-processing in the example's custom handler. Now, this will require good amount of effort to make it work with different sizes of the text and related audio output. 

Hence, as you have already indicated we will document this limitation of the waveglow example with reference to above ticket. If required, we can track this enhancement through a different ticket.

Reply:
@harshbafna This should be fine. Please describe the current limitation is the doc and then merge the PR